SRPTests/RCLinkTests.cs

```diff
@@ -0,0 +1,113 @@
+using Markdig;
+using NUnit.Framework;
+using ScriptureRenderingPipeline.Helpers.MarkdigExtensions;
+using ScriptureRenderingPipeline.Renderers;
+using System.Collections.Generic;
+
+namespace SRPTests
+{
+    public class RCLinkTests
+    {
+        private RCLinkOptions options;
+        private MarkdownPipeline pipeline;
+
+        [SetUp]
+        public void Setup()
+        {
+            this.options = new RCLinkOptions()
+            {
+                BaseUser = "WycliffeAssociates",
+                // TODO: this needs to be changed to a configuration value

```

Comment:
This todo is technically false since this is in a test.

Reply:
You're right.  Fixed.